Assets/MixedRealityToolkit.SDK/Features/Input/Handlers/SpeechInputHandler.cs

```diff
@@ -108,9 +133,24 @@ void IMixedRealitySpeechHandler.OnSpeechKeywordRecognized(SpeechEventData eventD
             {
                 keywordResponse.Invoke();
                 eventData.Use();
+
+                // Instantiate the Speech Confirmation Tooltip prefab if assigned
+                // Ignore "Select" keyword since OS will display the tooltip. 
+                if ((SpeechConfirmationTooltipPrefab != null) && (speechConfirmationTooltipPrefabInstance == null) && (eventData.Command.Keyword.ToLower() != "select")) 
```

Comment:
Nit: Maybe have a string array of commands to ignore that is populated with 'select' by default?

Reply:
@Railboy Oh... it might be better to actually scrape the list of the system reserved keywords dynamically. Might be better to work on it as a separate issue/PR. I am not sure if we have a good way to access the list of system keywords.